lua/fastaction/window.lua

```diff
@@ -108,7 +108,7 @@ function M.popup_window(content, on_buf_create, opts)
 
     local line = 2 -- avoid the title and the divider i.e. start at line 2
     for _, _ in pairs(content) do
-        vim.api.nvim_buf_add_highlight(buffer, m.namespace, 'MoreMsg', line, 0, 4)
+        vim.api.nvim_buf_add_highlight(buffer, m.namespace, 'MoreMsg', line, 0, 3 + opts.chars)
```

Comment:
instead of adding an option, It would be better to calculate the width based on #content - 2, i.e. height - 2 // (26 - #m.dismiss_keys). And even from dismiss keys only alphabets.